torchao/quantization/quant_api.py

```diff
@@ -1594,8 +1592,12 @@ class Float8DynamicActivationFloat8WeightConfig(AOBaseConfig):
     weight_dtype: torch.dtype = e4m3_dtype
     granularity: Optional[Union[FP8Granularity, List[FP8Granularity]]] = None
     mm_config: Optional[Float8MMConfig] = None
+    activation_scale_ub: Optional[float] = None
+    kernel: str = "aten"
     set_inductor_config: bool = True
 
+    _SUPPORTED_KERNELS = ["aten", "fbgemm"]
```

Comment:
imo we can design this a bit more:
1. make an enum?
2. should it say "backend" instead of "kernel"?

Reply:
yeah sure